packages/thumbprint-react/components/Tooltip/index.module.scss

```diff
@@ -1,12 +1,23 @@
 @import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
 
+@keyframes tooltip-animation {
+    0% {
+        opacity: 0;
+    }
+
+    100% {
+        opacity: 1;
+    }
+}
+
 $tooltip-max-width: 235px;
 
 .childrenWrapper {
     display: inline-flex;
 }
 
 .tooltip {
+    animation: tooltip-animation $tp-duration__2;
```

Comment:
Should this also have easing, or is that only for transitions?

Reply:
It uses the browser default if nothing's is provided but, yes, no reason not to use a token here.